crypto/x509/x509_lu.c

```diff
@@ -294,25 +294,22 @@ X509_LOOKUP *X509_STORE_add_lookup(X509_STORE *v, X509_LOOKUP_METHOD *m)
     }
 }
 
-X509_OBJECT *X509_STORE_get_X509_by_subject(X509_STORE_CTX *vs, int type,
-                                                  X509_NAME *name)
+X509_OBJECT *X509_STORE_CTX_get_obj_by_subject(X509_STORE_CTX *vs, int type,
```

Comment:
Please change the prefix from `X509_STORE_CTX_` to `X509_STORE_`.  You're not manipulating the ctx, nor are you retrieving values from it, you'r just using it to get information from the associated store, right?